src/import-data/ImportData.ts

```diff
@@ -87,119 +103,133 @@ export const ImportData = () => {
     loadTemplateWorkspaces();
   });
 
-  const importPFB = (namespace, name) => {
-    return async () => {
-      const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(url, 'pfb', null);
-      asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId }));
-      notifyDataImportProgress(jobId);
-    };
+  const importPFB = async (importRequest: PFBImportRequest, workspace: WorkspaceInfo) => {
+    const { namespace, name } = workspace;
+    const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(importRequest.url, 'pfb', null);
```

Comment:
I wonder if we should add `Workspaces.workspace(WorkspaceInfo)`, if this is a common case.

Reply:
That would definitely make some things more convenient. I didn't want to get into changes to the Ajax functions in this PR though.